CHANGELOG.md

```diff
@@ -209,6 +209,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Fixes management app breadcrumb error ([#2344](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2344))
 - [BUG] Fix suggestion list cutoff issue ([#2607](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2607))
 - [TSVB] Fixes undefined serial diff aggregation documentation link ([#3503](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3503))
+- Fix error message display issue in dev tool console ([#3652](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3652))
```

Comment:
is this issue introduced with new dev tool change for MD? if so, assume previous version doesn't has such issue, do we still need to callout and add to changelog?

Reply:
It's introduced by the dev tool refactor PR, literally it's not even related to MD. It's a good point, previous version doesn't have issue. Currently it's only that each PR is mapped to at least one changelog. 

Any thoughts? @kavilla @abbyhu2000

Reply:
yep, @seraphjiang is correct - if we're only fixing an unreleased bug, no need for a new changelog entry. In some cases we would maybe want to update the original changelog entry, but in this case, I'd just skip.

And when we generate the release notes, we'll want to do some additional cleanup and consolidation, so that all the MD entries are grouped and nicely formatted.

Reply:
removed change log item